packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -134,6 +134,21 @@ bool debugPrintLayouts = false;
 /// enabled by the test framework.
 bool debugCheckIntrinsicSizes = false;
 
+/// Adds [dart:developer.Timeline] events for every [RenderObject] layout.
+///
+/// For details on how to use [dart:developer.Timeline] events in the Dart
+/// Observatory to optimize your app, see:
+/// <https://fuchsia.googlesource.com/topaz/+/master/shell/docs/performance.md>
+///
+/// See also:
+///
+///  * [debugPrintLayouts], which does something similar for layout but using
+///    console output.
+///  * [debugProfileBuildsEnabled], which does something similar for widgets
```

Comment:
Can you please also update the documentation of these two flags to include a link back to this new flag?

Reply:
With my pleasure:)